.github/workflows/tests.yml

```diff
@@ -0,0 +1,100 @@
+name: Run CI tests
+
+# Controls when the action will run. 
+on:
+  # Triggers the workflow on push or pull request events but only for the master branch
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
```

Comment:
I'm not clear why we're limiting these here. I'd say we want the tests to run on all branches, no? A PR to a PR for instance, should still run the tests.

Reply:
I started with the example template, so these are the default values. I assume we should just use *?

Reply:
The easiest thing is to eliminate the filter entirely. I'll add another comment suggestion that's committable.

Reply:
Oh, you already got it! 😸